lib/fog/openstack/volume_v1.rb

```diff
@@ -32,22 +35,24 @@ class V1 < Fog::Volume::OpenStack
         request :list_volumes
         request :list_volumes_detailed
         request :create_volume
+        request :update_volume
         request :get_volume_details
         request :extend_volume
         request :delete_volume
 
         request :list_zones
 
         request :list_volume_types
-        request :create_volume_type
```

Comment:
What's happening with this request? Why is it disappearing?
Same for following one.